quickstarts/android/01-login.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Auth0 Android Quickstarts - 1. Login
```

Comment:
I think @ntotten mentioned only leaving the main title here (in this case: "Login"). No environment references, no numbers, name only.